deploy/aws/eks/templates/kubeconfig.tpl

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
I think we should avoid creating this file ourselves. There is an AWS command for this: https://docs.aws.amazon.com/cli/latest/reference/eks/update-kubeconfig.html

Reply:
Thanks for the reference! However, this is behavior of the original EKS module https://github.com/terraform-aws-modules/terraform-aws-eks/blob/80b0bc79af5d1218490d550d1c63f483e4948ee1/data.tf#L46, and we hope that the changes can be minimized

Reply:
:ok_hand:

Reply:
Created an issue for this: https://github.com/pingcap/tidb-operator/issues/621

Reply:
I've removed the customized eks module, now the kubeconfig is generated by the canonical eks module... in the same way😂